src/main/scala/com/johnsnowlabs/nlp/annotators/common/TokenizedWithSentence.scala

```diff
@@ -24,7 +24,7 @@ object TokenizedWithSentence extends Annotated[TokenizedSentence] {
         token.begin >= sentence.start & token.end <= sentence.end
       ).map(token => IndexedToken(token.result, token.begin, token.end))
       sentenceTokens
-    }).filter(_.nonEmpty).zipWithIndex.map{case (indexedTokens, index) => TokenizedSentence(indexedTokens, index)}
```

Comment:
Wait, i am not completely convinced, not sure if we want empty tokenized sentences here. Can't we remove the `zipWithIndex` and find out the original index from elsewhere?